crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/MaatCourtDataService.java

```diff
@@ -20,7 +20,7 @@ public class MaatCourtDataService {
     private final MaatCourtDataClient maatCourtDataClient;
     public static final String RESPONSE_STRING = "Response from Court Data API: %s";
 
-    public IOJAppealDTO getCurrentPassedIOJAppealFromRepId(Integer repId, String laaTransactionId) {
+    public IOJAppealDTO getIOJAppeal(Integer repId, String laaTransactionId) {
```

Comment:
This could be misleading as we are only retrieving the latest passed IOJ appeal.

Reply:
Good point, I thought we were just using the find endpoint. I'll change this back for now, but I think we should probably consider unifying the endpoints at some point. It looks like we could just pass an optional decisionResult parameter instead.